ledger/ledger-utxo/src/main/kotlin/net/corda/v5/ledger/utxo/UtxoLedgerService.kt

```diff
@@ -0,0 +1,56 @@
+package net.corda.v5.ledger.utxo
+
+import net.corda.v5.base.annotations.DoNotImplement
+import net.corda.v5.base.annotations.Suspendable
+import net.corda.v5.ledger.utxo.transaction.UtxoTransactionBuilder
+
+/**
+ * Defines UTXO ledger services.
+ */
+@DoNotImplement
+interface UtxoLedgerService {
+
+    /**
+     * Gets a UTXO transaction builder
+     *
+     * @return Returns a new [UtxoTransactionBuilder] instance.
+     */
+    @Suspendable
+    fun getTransactionBuilder(): UtxoTransactionBuilder
+
+    /**
+     * Resolves the specified [StateRef] instances into [StateAndRef] instances of the specified [ContractState] type.
+     *
+     * @param T The underlying [ContractState] type.
+     * @param stateRefs The [StateRef] instances to resolve.
+     * @return Returns a [List] of [StateAndRef] of the specified [ContractState] type.
+     */
+    @Suspendable
+    fun <T : ContractState> resolve(stateRefs: Iterable<StateRef>): List<StateAndRef<T>>
```

Comment:
I'd be keen for some of these methods to be moved to another service at some point in the future.

Reply:
Eventually this service is going to be huge if everything gets put onto it.